decidim-dev/decidim-dev.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |s|
   s.add_dependency "launchy"
   s.add_dependency "i18n-tasks", "0.9.8"
   s.add_dependency "faker", "~> 1.7.2"
-  s.add_dependency "poltergeist", "~> 1.12.0"
+  s.add_dependency "poltergeist", "~> 1.13.0"
   s.add_dependency "rails-controller-testing", "~> 1.0.1"
   s.add_dependency "simplecov", "~> 0.12"
```

Comment:
I updated simplecov to 0.13 in #629, but forgot to bump this number here, can you do it?